packages/crawler/src/apify-resources/resource-creator.ts

```diff
@@ -2,13 +2,12 @@
 // Licensed under the MIT License.
 import Apify from 'apify';
 
-export interface CrawlerFactory {
+export interface ResourceCreator {
```

Comment:
Too generic name, rename to ApifyResourceCreator?

Reply:
This is the interface, that's why it's generic, the implementation of it in the same file is ApifyResourceCreator which uses apify resources, later we can use something like MemoryResourceCreator for using the memory as a resource